tc/core/polyhedral/schedule_isl_conversion.cc

```diff
@@ -237,6 +237,60 @@ isl::schedule toIslSchedule(const ScheduleTree* root) {
 
 namespace {
 
+std::unique_ptr<ScheduleTreeElemBand> fromIslScheduleNodeBand(
+    isl::schedule_node_band b) {
+  auto res =
+      ScheduleTreeElemBand::fromMultiUnionPwAff(b.get_partial_schedule());
+  res->permutable_ = b.get_permutable();
+  for (size_t i = 0; i < b.n_member(); ++i) {
+    res->coincident_[i] = b.member_get_coincident(i);
+  }
+  return res;
+}
+
+std::unique_ptr<ScheduleTreeElemBase> elemFromIslScheduleNode(
```

Comment:
I find that the `fromXXX` nomenclature is better for static class members while I prefer `makeYYY(const XXX&)` for free functions. I think we have generally been consistent with `std::unique_ptr<YYY> makeYYY(...)` (except apparently for `fromIslScheduleNodeBand` above). 

I would use `makeElemBand` and `makeElem` instead, or something similar.

Reply:
Except that we use `make*` for static member functions in this codebase.
`makeElem` already exists, but ok, I'll rename this one.

Reply:
After looking in more detail, I think I keep the "from*" notation.  These are functions _local_ to a file, and almost every other function is that file is called "fromIsl*" (except for a handful of "toIsl*" that explains the choice of "from"/"to").